lib/nightmare.js

```diff
@@ -80,8 +80,9 @@ function Nightmare(options) {
 
   process.setMaxListeners(Infinity);
   process.on('uncaughtException', function(err) {
-    console.error(err.stack);
+    debug(err.stack);
```

Comment:
Good point, however that means any `process.on('uncaughtException',` that the user has added might not get called. However I've just tested it and neither does re-throwing! Looks like Node handles an error inside an `uncaughtException` handler as a special case. I've switched to `console.error` and `process.exit(1)`.